File: 백준/Bronze/2920. 음계/음계.java
import java.util.Arrays;
import java.util.Scanner;
import java.util.Collections;

public class Main { 
    public static void main(String[] args) { 
        int[] num = new int[8];

        int checkA = 0;
        int checkD = 0;

        Scanner sc = new Scanner(System.in);
        for(int i=0;i<8;i++) {
            num[i] = sc.nextInt();
        }

        for(int i=0;i<8;i++) {
            if(num[i] == i+1) checkA++;
            else if(num[i] == 8-i) checkD++;
        }

        if(checkA == 8) System.out.println("ascending");
        else if(checkD == 8) System.out.println("descending");
        else System.out.println("mixed");
    } 
}

File: 백준/Bronze/2920. 음계/음계.py
numList = list(map(int, input().split()))
ascCheck=1
descCheck=8
asc=False
desc=False

if numList[0]==1:
    asc = True
if numList[0]==8:
    desc = True

for num in numList:
    if asc == True:
        if num == ascCheck:
            ascCheck = ascCheck+1
        else:
            asc = False
            break
    if desc == True:
        if num == descCheck:
            descCheck = descCheck-1
        else:
            desc = False
            break

if asc == True:
    print("ascending")
elif desc == True:
    print("descending")
else:
    print("mixed")